src/shared/sidepanel/sidepanel-support.ts

```diff
@@ -0,0 +1,3 @@
+export function isSidepanelSupported() {
+  return globalThis.chrome && 'sidePanel' in chrome;
```

Comment:
`&& chrome.sidePanel != null`

Reply:
This is a canonical way to check for browser apis
There's no reason for the property to exist and be nullish, I think. And we're not checking deeper anyway